packages/css/src/heading/README.md

```diff
@@ -0,0 +1,23 @@
+# Heading
+
+Een heading zegt iets over de onderstaande content. Met behulp van headings kan de gebruiker snel een overzicht krijgen van de structuur van de webpagina.
+
+## Richtlijnen
+
+- Een heading omschrijft de content eronder.
+- Gebruik een heading niet voor opmaak en styling, maar om de structuur van de pagina weer te geven.
+- Gebruik headings hierarchisch en sla geen heading niveaus over. Dus na een `h1` volgt een `h2` en dus niet een `h3`.
+
+## Relevante WCAG eisen
+
+- [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/quickref/#qr-content-structure-separation-programmatic):
+  Informatie, structuur en relaties die via presentatie worden overgebracht, programmatisch bepalen of in tekst aanbieden
```

Comment:
In my PR I chose to describe what part of the WCAG rules are relevant to that component, you've chosen to write a generic description. What do we prefer?

An advantage of a generic description is that they're reusable for other components. A disadvantage is that it's harder to figure out in what way the description relates to the component. What does the text above mean for the Heading component?

Reply:
Made it more specific for heading. I think we need to make it readable for humans. WCAG can be very technical en theoretical.